.eslintrc.js

```diff
@@ -2,29 +2,107 @@ module.exports = {
   env: {
     browser: true,
     es2020: true,
-    node: true
+    node: true,
   },
   extends: [
-    'eslint:recommended',
-    'plugin:react/recommended',
-    'plugin:jest/recommended',
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:jest/recommended",
```

Comment:
Is there a reason this is the only file with double quotes used instead of single quotes? Every other file seems to have been changed to use single quotes - whereas this one has been changed to use double quotes.

Reply:
I think the rest of the files changed to single quotes due to the autofix, but does that convention not apply to this file for a particular reason?

Reply:
That's a good point! Currently, the path for this file isn't hit by the glob for `npm run lint-js-fix`, lemme change this one manually.